src/routes/JobCreateKBARTRoute.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import compose from 'compose-function';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import { stripesConnect } from '@folio/stripes/core';
+import withFileHandlers from './components/withFileHandlers';
+import View from '../components/views/JobForm';
+
+class JobCreateKBARTRoute extends React.Component {
+  static manifest = Object.freeze({
+    jobs: {
+      type: 'okapi',
+      path: 'erm/jobs/kbartImport',
+      fetch: false,
+      shouldRefresh: () => false,
+    },
+  });
+
+  static propTypes = {
+    handlers: PropTypes.object,
+    history: PropTypes.shape({
+      push: PropTypes.func.isRequired,
+      replace: PropTypes.func,
+    }).isRequired,
+    location: PropTypes.shape({
+      search: PropTypes.string.isRequired,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      jobs: PropTypes.shape({
+        POST: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
+  };
+
+  static defaultProps = {
+    handlers: {},
+  }
+
+  handleClose = () => {
+    const { location } = this.props;
+    this.props.history.push(`/local-kb-admin${location.search}`);
+  }
+
+  handleSubmit = (job) => {
+    const { history, location, mutator } = this.props;
+    let jobName = '';
+    let jobId = '';
+    let jobClass = '';
+
+    mutator.jobs
+      .POST(job)
+      .then(response => {
+        jobName = get(response, 'name', '');
```

Comment:
Also, while you are at it, can you run through this PR and replace all the lodash `get`s with the new `?.` optional chaining logic. We use it [here](https://github.com/folio-org/ui-agreements/blob/master/src/routes/AgreementCreateRoute.js#L153) for your reference

Reply:
I was planning to do that, but how does one set a default value for that? `get` seemed cleaner whenever I wanted say an empty object or string as default, rather than say:

`response?.name ? response?.name : {}` or `response?.name ? response?.name : ''`

Reply:
Ah right, as far as I can tell, the way to do this is with the nullish operator:
```data={{
            jobs: resources?.jobs?.records ?? [],
            resultValues: resources?.resultValues?.records ?? [],
            statusValues: resources?.statusValues?.records ?? [],
          }}``` I don't like the way this looks, but I'll use it for now